plutus-use-cases/src/Language/PlutusTx/Coordination/Contracts/Vesting.hs

```diff
@@ -136,13 +144,20 @@ validatorScriptHash =
     . Ledger.scriptAddress
     . validatorScript
 
+{-# INLINABLE mkValidator #-}
 mkValidator :: Vesting -> VestingData -> () -> PendingTx -> Bool
-mkValidator d@Vesting{..} VestingData{..} () p@PendingTx{pendingTxValidRange = range} =
+mkValidator d@Vesting{..} VestingData{..} () ptx@PendingTx{pendingTxValidRange = range} =
     let
+        payBack :: Value
+        payBack = mconcat (map pendingTxOutValue (getContinuingOutputs ptx))
+
+        lockedValue :: Value
+        lockedValue = pendingTxInValue (pendingTxIn ptx)
+
         -- We assume here that the txn outputs are always given in the same
         -- order (1 PubKey output, followed by 0 or 1 script outputs)
```

Comment:
This comment is inaccurate.

Reply:
Changed the comment, and added some more.